packages/grpc/src/internal/client.ts

```
@@ -73,15 +74,32 @@ class ClientBase implements IRequireInitialization, IDisposable {
 }
 
 export function createGrpcClient<T>(
-    config: IGrpcClientConfiguration & IGrpcConfiguration
+    config: IGrpcClientConfiguration & IGrpcConfiguration,
+    certPath?: string
 ): T & IDisposable & IRequireInitialization {
     const serviceDef = createServiceDefinition(config.definition);
+    let client: Client;
     const ClientType = makeGenericClientConstructor(serviceDef, undefined, undefined);
-    const client = new ClientType(
-        config.endpoint,
-        credentials.createInsecure(),
-        createGrpcConfiguration(config)
-    );
+    if (certPath) {
+        const rootCert = readFileSync(certPath);
```

will this cert logic flow throw error ?

---

readFileSync should throw an error if file doesn't exist for example.